oap-server/server-starter/src/main/resources/ui-initialized-templates/general/general-service.json

```diff
@@ -860,6 +860,19 @@
                   "type": "Log"
                 }
               ]
+            },
+            {
+              "name": "Slow Statements",
+              "children": [
+                {
+                  "x": 0,
+                  "y": 0,
+                  "w": 24,
+                  "h": 48,
+                  "i": "0",
+                  "type": "SlowStatements"
+                }
+              ]
```

Comment:
This should bind an expression like I shared in https://github.com/apache/skywalking/pull/13514#issuecomment-3326366693

Reply:
Nothing should be hard-coded.